docs/source/howto/run_codes.rst

```diff
@@ -346,6 +347,28 @@ At the end, you receive a confirmation, with the *PK* and the *UUID* of your new
 
     Note: remove the ``--`` prefix and replace dashes (``-``) within the keys with an underscore ( ``_`` ).
 
+
+.. note::
+
+    It is possible to run codes that are provided by a `Conda environment <https://docs.conda.io/en/latest/>`_.
```

Comment:
Perhaps start with the computer configuration
```suggestion
    It is possible to run codes that are provided by a `Conda environment <https://docs.conda.io/en/latest/>`_.
    First, configure the AiiDA computer with ``--use-login-shell`` to enable access to the ``conda activate`` command.
```

Reply:
Moved the computer configuration to the top. But `--use-login-shell` is not the correct option. It it the `shebang` that should be set to `#!/bin/bash -l`